packages/blade/src/components/Tooltip/Tooltip.web.tsx

```diff
@@ -98,7 +98,7 @@ const Tooltip = ({
       {React.cloneElement(children, {
         ref: refs.setReference,
         ...mergeProps(children.props, getReferenceProps()),
-        ...makeAccessible({ label: content }),
+        ...makeAccessible({ describedBy: content }),
```

Comment:
did you mean `description` here? `describedby` expects id of the description no?

Also why this change?

<img width="802" alt="image" src="https://github.com/razorpay/blade/assets/30949385/cb3dd2ea-9f28-479d-9bb3-48f37d2e5d60">

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby#example

Reply:
Oh right, it should not be the content

Reply:
Yeah, I updated the logic not to override the aria-label if provided by the user. We're not adding `aria-description` as [recommended in the docs.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description)

@anuraghazra @saurabhdaware